src/test/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidationTest.java

```diff
@@ -0,0 +1,103 @@
+
+    ObjectMapper om = new ObjectMapper();
+    OfficersApi mockOfficers = new OfficersApi();
+    CompanyProfileApi companyProfileApi = new CompanyProfileApi();
+
+    @Mock
+    private OfficerService officerService;
+
+    @InjectMocks
+    private CompanyOfficerValidation companyOfficerValidation;
+
+    @BeforeEach
+    void init() throws JsonProcessingException {
+        OFFICER_LIST.clear();
+        CompanyOfficerApi MOCK_OFFICER = om.readValue(DIRECTOR_JSON, CompanyOfficerApi.class);
+        OFFICER_LIST.add(MOCK_OFFICER);
+        companyProfileApi.setCompanyNumber(COMPANY_NUMBER);
+    }
+
+    @Test
+    void validateDoesNotThrowOnSingleOfficerCompany() throws ServiceException {
+        mockOfficers.setItems(OFFICER_LIST);
+        mockOfficers.setActiveCount((long) OFFICER_LIST.size());
+
+        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(mockOfficers);
+
+       assertDoesNotThrow(() -> companyOfficerValidation.validate(companyProfileApi));
+    }
+
+    @Test
+    void validateThrowsOnMultipleOfficerCompany() throws JsonProcessingException, ServiceException {
+        CompanyOfficerApi director = om.readValue(DIRECTOR_JSON,CompanyOfficerApi.class);
+        OFFICER_LIST.add(director);
+        mockOfficers.setItems(OFFICER_LIST);
+        mockOfficers.setActiveCount((long) OFFICER_LIST.size());
+
+        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(mockOfficers);
+
+        var ex = assertThrows(EligibilityException.class, () ->
+                companyOfficerValidation.validate(companyProfileApi));
+
+        assertEquals(EligibilityFailureReason.INVALID_OFFICER_COUNT,ex.getEligibilityFailureReason() );
+    }
+
+    @Test
+    void validateDoesNotThrowOnSingleOfficerCompanyWithSecretaries() throws JsonProcessingException, ServiceException {
+        CompanyOfficerApi secretary = om.readValue(SECRETARY_JSON,CompanyOfficerApi.class);
+        OFFICER_LIST.add(secretary);
+        mockOfficers.setItems(OFFICER_LIST);
+        mockOfficers.setActiveCount((long) OFFICER_LIST.size());
+
+        when(officerService.getOfficers(COMPANY_NUMBER)).thenReturn(mockOfficers);
+
+        assertDoesNotThrow(() -> companyOfficerValidation.validate(companyProfileApi));
+    }
+
+    @Test
+    void getOfficerCountReturnsNumberOfOfficersExcludingSecretaries() throws JsonProcessingException {
+        CompanyOfficerApi director = om.readValue(SECRETARY_JSON,CompanyOfficerApi.class);
+        CompanyOfficerApi secretary = om.readValue(DIRECTOR_JSON,CompanyOfficerApi.class);
+        OFFICER_LIST.add(director);
+        OFFICER_LIST.add(secretary);
+        mockOfficers.setItems(OFFICER_LIST);
+        mockOfficers.setActiveCount((long) OFFICER_LIST.size());
+
+        var result = companyOfficerValidation.getOfficerCount(mockOfficers.getItems(), mockOfficers.getActiveCount());
+        assertEquals(result, 2L);
+        assertNotEquals(result, OFFICER_LIST.size());
```

Comment:
these two have their arguments the wrong way around.
It should be `assert(expected, actual)` so in this case `assertEquals(2L, result)`